archesproject/arches
# adds ENABLE_USER_SIGNUP and FORCE_USER_SIGNUP_EMAIL_AUTHENTICATION #4936

### Types of changes


### Description of Change

Adds ENABLE_USER_SIGNUP and FORCE_USER_SIGNUP_EMAIL_AUTHENTICATION settings.

To test ENABLE_USER_SIGNUP:
- set ENABLE_USER_SIGNUP to False
- notice the signup link in the login template is hidden
- access the signup page manually (http://localhost:8000/auth/signup)
- notice error thrown

To test FORCE_USER_SIGNUP_EMAIL_AUTHENTICATION:
- set FORCE_USER_SIGNUP_EMAIL_AUTHENTICATION to False
- create a new user
- notice how user is automatically redirected to auth with message


### Issues Solved

#4936
